config/config.go

```diff
@@ -37,6 +37,9 @@ const (
 
 	// DefaultServerTLSKey is the default path to use when looking for the Buildkit TLS key
 	DefaultServerTLSKey = "./certs/buildkit_key.pem"
+
+	// DefaultContainerFrontend is the default frontend program or interfacing with the running containers and saved images
+	DefaultContainerFrontend = "docker-shell"
```

Comment:
Would it be safe to use `"auto"` here already?

Reply:
Probably, was wanting to minimize the number of new behaviors shipped. If youre comfortable, I am

Reply:
Yeah.. it's probably safest to be turned off completely by default given that we'll start in experimental status.